### ISSUE TITLE
Improve story for creating PRs to update to nightly builds

### ISSUE THREAD

**Issue by felixfbecker** [renovatebot/renovate#5872]
**What would you like Renovate to be able to do?**
I'd like to use Renovate to open PR for select packages that publish nightly builds, e.g. TypeScript or eslint-typescript. This way I can check builds and see if the next version will break anything, and can open issues when they crash (potentially even automatically).

At the same time, I'd still like PRs for upgrades to the `latest` version.

Currently,
- I can only track one dist-tag
- Tracking a nightly dist-tag always creates a second PR "Rollback foo to x.y.z": https://github.com/sourcegraph/sourcegraph/pulls?q=is%3Apr+is%3Aopen+sort%3Aupdated-desc+in%3Atitle+%22dependency+typescript%22+author%3Aapp%2Frenovate+

**Describe the solution you'd like**
Allow configuration to track multiple dist-tags, e.g. `latest` and `next`.

**Describe alternatives you've considered**

**Additional context**

**Comment by rarkins** [comment]
I don’t think it completely satisfies your requirement but how dues this compare to`followTag`?

**Comment by felixfbecker** [comment]
`followTag` is basically what I want, but I need it to be an array to accept multiple tags. Right now, if I set it to `nightly`, I won't get any PRs more to upgrade to stable versions :/
